includes/Admin/Admin.php

```diff
@@ -76,6 +76,7 @@ public function register(): void {
 		add_filter( 'default_content', [ $this, 'prefill_post_content' ], 10, 2 );
 		add_filter( 'default_title', [ $this, 'prefill_post_title' ] );
 		add_filter( 'display_media_states', [ $this, 'media_states' ], 10, 2 );
+		add_filter( 'web_stories_erase_data_on_uninstall', [ $this, 'data_removal' ], 10, 2 );
```

Comment:
Instead of filtering `web_stories_erase_data_on_uninstall`, we should just call `get_option( Settings:: SETTING_NAME_DATA_REMOVAL )` in `uninstall.php` directly.

This is because the `Admin` class here is only loaded in the admin, but a plugin can also be uninstall via WP-CLI for example.

Note: Using `get_option` should be fine, but could theoretically also use the Services injector, though seems a bit overkill. Plus not sure if it's properly available in CLI context as well, so needs some testing. Hence the suggestion to just use `get_option`.

cc @spacedmonkey